drafthorse/pdf.py

```diff
@@ -38,10 +37,21 @@
     create_string_object,
 )
 
+from drafthorse.xmp_schema import XMP_SCHEMA
+
+logging.basicConfig()
 logger = logging.getLogger("drafthorse")
+logger.setLevel(logging.INFO)
 
 
-def attach_xml(original_pdf, xml_data, level="BASIC"):
+def attach_xml(original_pdf, xml_data):
```

Comment:
I think we should keep the `level` parameter and just change the default to `None` and then perform autodetection (see below for one reason).